### PR TITLE
use Python 3.9

### PR DESCRIPTION
**Pull request by goneri**
Upgrade the default Python version. Run the `policy` check by default now we've got a Python version that is compatible with an up to date `ansible-core` release.

### CODE REVIEW

**Comment by goneri** [comment]
Needs https://github.com/mattclay/aws-terminator/pull/248 first.

File: tox.ini [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tox]
 skipsdist=True
-envlist=pycodestyle,pylint,yamllint
+envlist=pycodestyle,pylint,yamllint,policy
```
Comment (mattclay) [inline tox.ini:3]: Is this change related to the Python 3.9 change?

Comment (goneri) [reply tox.ini:3]: Yes, we were not able to run the policy check anymore with Ubuntu18.04/Python3.7. I will mention this change in the PR description.

Comment (goneri) [reply tox.ini:3]: Ok, this raise a problem with community.general. I will isolate this change in a new PR.

**Comment by mattclay** [comment]
@goneri I don't expect there to be any issues upgrading to Python 3.9, but it would be a good idea to look at the generated Lambda zip files to make sure there are no unexpected new files. I also recommend reviewing the logs once this is deployed to make sure no unexpected issues show up.

File: tox.ini (hunk 1 above)
Comment (mattclay) [reply tox.ini]: I'm curious, how does this change have any effect on community.general?

Comment (goneri) [reply tox.ini]: This was the error:

```
TASK [Fail if policy is too large] ***********************************************************************************************************************************************************************************************************
[WARNING]: an unexpected error occurred during Jinja2 environment setup: unable to locate collection community.general
fatal: [localhost]: FAILED! => {"msg": "An unhandled exception occurred while templating '{{ policies | json_query(_query) | list }}'. Error was a <class 'ansible.errors.AnsibleError'>, original message: template error while templating string: unable to locate collection community.general. String: {{ policies | json_query(_query) | list }}"}
```

See: https://github.com/mattclay/aws-terminator/pull/250

**Comment by goneri** [comment]
@mattclay I did the test in our team account (ca-central-1) before pushing this series of PR, it works fine.

**Review by mattclay — APPROVED** [review]

**Pull request merged**